src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -54,6 +54,11 @@ enums:
         meaning: rdfs:Datatype
       rdf property:
         meaning: rdf:Property
+      composed entity:
+        description: This value indicates that the entity ID does not represent a single entity, but a composite involving several individual entities. This value MUST NOT be used in the predicate_type slot. This specifications does not prescribe how an ID representing a composite entity should be interpreted; this is left at the discretion of applications.
```

Comment:
Do we prefer `composed entity` over something like `{composed,composite} entity expression`?

I wish there was an element in the OWL vocabulary we could map to. the OWLAPI has a notion of "AnonymousClassExpression" but that doesn't map to anything in the OWL spec (https://www.w3.org/TR/owl2-syntax/#Class_Expressions). And in any case, a composed entity need not be a class expression.

I think the OWLAPI's use of "anonymous" for compound expressions is quite unique here. The more typical mathematic adjective is "composed" or "compound" or "complex" AFAICT. But I think adding "expression" makes it clear this does not denote a named entity in the more typical sense?

Reply:
> Do we prefer composed entity over something like {composed,composite} entity expression?

I don’t really have a strong preference here. My main concerns with the choice of the name was (1) to convey the idea that it is not a “simple” entity and (2) to avoid tying the concept to any particular representation or encoding (such as the “URI Expression” proposal mentioned in the #402 ticket). If people think that the first aim is better achieved by adding ` expression`, I have no objection to that.